README.md

```diff
@@ -1,102 +1,114 @@
-Setting up for local development
-================================
+# Setting up for local development
 
-To work on hosted rapidpro:
+## 1. Create an ngrok account
 
+Use ngrok to set up a URL that fowards to your development server. You'll need
+a paid account so that you can set up a subdomain.
+
+1. Download the ngrok zip: https://ngrok.com/download
+
+2. Unzip the package, and move the `ngrok` executable to a convenient
+   location.
+
+3. Log into https://ngrok.com using the Google login with your Caktus account.
+   This will give you access to the "Caktus Group" organization, which is
+   a paid account.
+
+4. Configure ngrok to use the Caktus account.
+   Get the auth token here from https://dashboard.ngrok.com/auth
+
+    ```
+    ./ngrok authtoken <caktus-auth-token>
```

Comment:
Let's prefix this example shell command with `$` for consistency with other examples in the document.